charts/sonarqube-dce/Chart.yaml

```diff
@@ -79,6 +79,8 @@ annotations:
       description: "Support ApplicationNodes.extraVolumes and ApplicationNodes.extraVolumeMounts"
     - kind: added
       description: "Ensure kubernetes.io/version label is smaller than 63 chars"
+    - kind: changed
```

Comment:
Based on our release process, we should add this change on top of the list (after the chart bump). However, I'm not sure I agree with doing it, wdyt?

Reply:
Agree, resolved